test/run/test_stages.py

```diff
@@ -142,13 +142,15 @@ def _sorted_tree(tree):
         tree_diff2 = _sorted_tree(tree_diff2)
 
         def raise_assertion(msg):
+            text_file_diff = subprocess.check_output(
```

Comment:
I added this to help debug the authselect failure, we might consider to keep it though (should be in a separate PR though) as it might be useful in similar situations in the future.